src/main/java/org/opentripplanner/analyst/request/SampleGridRenderer.java

```diff
@@ -183,7 +188,7 @@ public WTWD closeSample(WTWD zUp, WTWD zDown, WTWD zRight, WTWD zLeft) {
                  * The computations below are approximation, but we are on the edge anyway and the
                  * current sample does not correspond to any computed value.
                  */
-                z.wTime = tMin + gridSizeMeters / v0;
+                z.wTime = tMin + gridSizeMeters / offRoadSpeed;
```

Comment:
I carried over the variable renaming when I did the merge. So I believe I took care of this problem and retained your changes.